pkg/elastic/elastic.go

```diff
@@ -8,7 +8,6 @@ import (
 	"fmt"
 	"io"
 	"os"
-        "strings"
         "crypto/tls"
```

Comment:
Can you check there are no extra spaces there, seems weird it looks with double tab, but no change reported. You can run `go fmt ./...` to fix any format issues.
Otherwise all looks good! :+1:

Reply:
Thanks, fixed it now.